src/pages/blog.tsx

```diff
@@ -1,101 +1,109 @@
+      listItem,
+      level,
+    },
+    }`;
 
-const Blog: NextPage = () => (
-  <Layout>
-    <Container>
-      <Hero
-        src={veltBoss}
-        alt="Picture of the boss in Velt"
-        quality={100}
-        placeholder="blur"
-      />
-      <Flex flexDirection="column" pt="m" gap="l">
-        <Flex flexDirection="column" px="xl" gap="xs">
-          <Text fontSize="3xl" color="text" lineHeight="9" fontWeight="bold">
-            Imitation; At copy economics, as large leaves communicated. White
-            been
-          </Text>
-          <Text
-            fontSize="m"
-            fontStyle="italic"
-            color="text"
-            fontWeight="hairline"
-          >
-            by Simen Pusegutt
-          </Text>
-        </Flex>
-        <Flex flexDirection="column" px="xl" gap="m">
-          <Text fontSize="xl" color="text">
-            Simple, can the and line could knowing end just far by feedback we
-            with were I or up it small something and was I gets every copy tried
-            with would however for accompany complete even seven desk were river
-            and a of one and the me. Can the so discharge.
-          </Text>
-          <Divider />
-          <Text color="text" fontWeight="medium" fontSize="l">
-            Sort always him, it be there long fortune. Rationally of to until
-            the and the again when on were class whom a of the would the
-            discharge a outfits learn defined once cities uninspired, subdued in
-            clearly, keep hand, nation project from spirits had of picked thinks
-            when redesigns. Too stupid the poets, mathematicians ship yards
-            checks, he heavy assistant other devious could was lay who through
-            got manipulate founding the fundamental; Counter. The and thought.
-            Well, on headline into one he greater, may the started commas,
-            dissolute still times back a and first is think, the this appointed
-            not.
-          </Text>
-          <Text color="text" fontWeight="medium" fontSize="l">
-            Goodness. Little target. Right to familiar on shown turned least
-            their the minutes. The one don&apos;t she then it while be one the
-            and to and of an if own often upright, know not windshield nature,
-            at that military to sort emerged. Have surprised was where a
-            collection richer to harmonics; The neighbours a they be people a
-            has he for the editorials disappointment in depend broad. To do.
-            Accompany head that of present gentlemen, of taken be applications
-            live were has, form would be ambushed day. And spirit, buttons it he
-            his key happens, to would a not, years.
-          </Text>
-        </Flex>
+const Blog = ({ posts }) => {
+  return (
+    <Layout>
+      <Flex flexDirection="column">
+        {posts.length > 0 && (
+          <Container>
+            {posts.map(post => (
```

Comment:
Don't think it makes sense to list the blog posts like this. IMHO, we should just get the first value of the array in this PR.

E.g.

```typescript
const [post] = posts;
```